remail/email_api/test_email_api.py

```
@@ -5,40 +5,16 @@
-
-exchange_test_email = Email(
-    subject="test_exchange_mail",
-    body="Test!!",
-    recipients=[
-        EmailReception(
-            contact=(Contact(email_address="thatchmilo35@gmail.com")),
-            kind=RecipientKind.to,
-        )
-    ],
+from exchangelib import (
+    Credentials,
```

Ruff (0.8.2)

`exchangelib.Credentials` imported but unused

Remove unused import

(F401)